eng/pipelines/templates/jobs/new.smoke.tests.yml

```diff
@@ -0,0 +1,176 @@
+parameters:
```

Comment:
similar to the old smoke.tests.yml, I put it up to test the changes/integration with Jesse's code. Won't be in the final version when merging. (main difference is the parameters passed to Update-Dependencies.ps1)